package.json

```diff
@@ -25,11 +26,15 @@
     "husky": "^0.14.3",
     "lint-staged": "^4.0.3",
     "prettier": "^1.5.3",
-    "request": "^2.72.0",
+    "request": "^2.81.0",
+    "request-promise": "^4.2.1",
```

Comment:
Non credo che serva più questa

Reply:
https://github.com/request/request-promise-native#installation
... 
  request is defined as a peer-dependency and thus has to be installed separately.

Reply:
Il commento si riferisce sempre alla riga subito sopra, quindi request-promise